packages/@aws-cdk/aws-dynamodb/test/test.dynamodb.ts

```diff
@@ -229,8 +229,165 @@ export = {
             }
         });
 
+        test.done();
+    },
+
+    'when specifying Read Auto Scaling'(test: Test) {
+        const app = new TestApp();
+        const table = new Table(app.stack, 'MyTable', {
+            tableName: 'MyTable',
+            readCapacity: 42,
+            writeCapacity: 1337
+        });
+        table.addPartitionKey('partitionKey', KeyAttributeType.String);
+        table.addSortKey('sortKey', KeyAttributeType.Binary);
+        table.configureReadAutoScaling({
+            minimumCapacity: 50,
+            maximumCapacity: 500,
+            scalingTargetValue: 75.0,
```

Comment:
What happens when this is over 100? both in this code and in cloudformation?

Reply:
Good point. We could add validation here, CDK can add value there.